moped-editor/src/views/dev/LookupsView/settings.js

```diff
@@ -176,4 +176,14 @@ export const SETTINGS = [
       },
     ],
   },
+  {
```

Comment:
@chiaberry since the list is getting long, what do you think about re-ordering these tables/nav buttons to be alphabetical?

- components
- component tags
- entities
- milestones
- phases
- tags

this works great and looks great! 🚢